week1/3-website/index.html

```diff
@@ -1,22 +1,113 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1, shrink-to-fit=no"
-    />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
```

Comment:
I like that you've done this, just because it's what I would do. It doesn't affect how the code works, but it just looks nicer, wouldn't you agree?

Reply:
Yes, I agree absolutely. Removed the spaces before the closing tags. Thank you @TomPaineCYF